prometheus/openshift/README.md

```diff
@@ -0,0 +1,37 @@
+## Prerequisites
+
+The following prerequisites must be met prior to beginning to deploy GitLab CE
+
+* 1 [Persistent Volume](https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/storage.html) or a cluster that supports [dynamic provisioning with a default StorageClass](https://docs.openshift.com/container-platform/latest/install_config/storage_examples/storage_classes_dynamic_provisioning.html)
+* OpenShift Command Line Tool
+* [Openshift Applier](https://github.com/redhat-cop/openshift-applier/) to deploy GitLab CE. As a result you'll need to have [ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html)
```

Comment:
`GitLab CE`

Reply:
I'd recommend pointing them to using the `openshift-applier` container run-time environment:
https://github.com/redhat-cop/openshift-applier#openshift-applier-container-image